testsuite/integration/elytron/pom.xml

```diff
@@ -51,13 +51,15 @@
         <jbossas.ts.dir>${jbossas.ts.integ.dir}/..</jbossas.ts.dir>
         <jbossas.project.dir>${jbossas.ts.dir}/..</jbossas.project.dir>
         <wildfly.dir>${basedir}/target/${wildfly.instance.name}</wildfly.dir>
-
+        <security.properties>${project.basedir}/src/test/resources/disabled_tlsv1.properties</security.properties>
```

Comment:
Since the properties file is used to re-enable TLSv1, it would be good to rename this to `reenabled_tlsv1.properties` or something like that.